### PR TITLE
Project design update (not flexible)

### PR DESCRIPTION
**Pull request by leonidbogin**
# Task - Index
_`Before`_
image
_`After`_
Task-Index

# Task - Edit
_`Before`_
image
_`After`_
Task-Edit

# Task - Details
_`Before`_
image
_`After`_
Task-Details

# Task - Delete
_`Before`_
image
_`After`_
Task-Delete

# CurrentTask - Index
_`Before`_
image
_`After`_
CurrentTasks-Index
`-----------------------------------------------------------------------------------`
# TaskTrack - Index
_`Before`_
image
_`After`_
TaskTrack-Index

# TaskTrack - Details
_`Before`_
image
_`After`_
TaskTrack-Details

# TaskTrack - Delete
_`Before`_
image
_`After`_
TaskTrack-Delete
`-----------------------------------------------------------------------------------`
# Member - Index
_`Before`_
image
_`After`_
Member-Index

# Member - Create
_`Before`_
image
_`After`_
Member-Create

# Member - Edit
_`Before`_
image
_`After`_
Member-Edit

# Member - Delete
_`Before`_
image
_`After`_
Member-Delete

### CODE REVIEW

File: DIMS-Core/DIMS-Core/Views/Task/Create.cshtml [hunk 1]
```diff
@@ -35,11 +35,16 @@
...
+            <div class="details-controls">
+                <div class="form-group">
+                    <input type="submit" value="Create" class="btn btn-primary" />
+                </div>
+                @await Html.PartialAsync("_BackToList", new BackToListViewModel
+                {
+                    Controller = ViewBag.BackController ?? "tasks",
+                    Action = ViewBag.BackAction
+                })
             </div>
```
Comment (maximsan) [inline DIMS-Core/DIMS-Core/Views/Task/Create.cshtml:47]: Is it possible to use the code as partial view? It's repeated in several places.

File: DIMS-Core/DIMS-Core/wwwroot/css/site.css [hunk 2]
```diff
@@ -137,10 +147,41 @@ body {
     min-width: 130px;
 }
 
+.table-index .col-id {
+    width: 70px;
```
Comment (maximsan) [inline DIMS-Core/DIMS-Core/wwwroot/css/site.css:151]: it's preferable to use relative values as rem instead of px

**Review by maximsan — CHANGES_REQUESTED** [review]
Looks fine for me except the question about repetitive code with back button. Can we improve it ?

File: DIMS-Core/DIMS-Core/Views/Task/Create.cshtml (hunk 1 above)
Comment (leonidbogin) [reply DIMS-Core/DIMS-Core/Views/Task/Create.cshtml]: Done.
https://github.com/Dev-incubator/DIMS-Core/pull/26/commits/64fe4dd8de672e1e0ee03116700d34166e1c0b30
https://github.com/Dev-incubator/DIMS-Core/pull/26/commits/f85978b6736cc350354064f1650ccfe243ec73cc

File: DIMS-Core/DIMS-Core/wwwroot/css/site.css (hunk 2 above)
Comment (leonidbogin) [reply DIMS-Core/DIMS-Core/wwwroot/css/site.css]: Done.
https://github.com/Dev-incubator/DIMS-Core/pull/26/commits/a203e9ce4d1177a5b98efa093245f28fb27496d7

**Review by PushMeNow — APPROVED** [review]
Approve

**Pull request merged**